test/test_suite_main_testcases.hpp

```diff
@@ -792,6 +821,90 @@ PRINTF_TEST_CASE(floating_point_specifiers_precision_and_flags)
   PRINTING_CHECK("3.5",                     ==, sprintf_, buffer, "%.1f", 3.49);
   PRINTING_CHECK("a0.5  ",                  ==, sprintf_, buffer, "a%-5.1f", 0.5);
   PRINTING_CHECK("a0.5  end",               ==, sprintf_, buffer, "a%-5.1fend", 0.5);
+
+  /* %lf for float that prompt to double */
```

Comment:
Again, no sense in using single-precision float literals, they just get promoted.

Reply:
It's just a test-case, not about the sense